cmd/app/command/start.go

```diff
@@ -46,7 +47,8 @@ func NewCmdStart() *cobra.Command {
 			metrics.StartingTime = time.Now()
 			// Start HTTP server for /metrics endpoint
 			go server.StartHttpServer()
-			controller.Watch(options.kubeconfig)
+			probe.Start()
```

Comment:
Does this call the probeInterface Start() ? If yes, why is this required?

Reply:
This is not calling `probeInterface Start()` directly this is required to invoke `init()` of each probe which calls `registerProbe()` and in `registerProbe()` `start()` is called if probe status is enabled.

Reply:
Can you put a comment above this call similar to StartHttpServer, on what proble.Start will accomplish.

Reply:
yes, added.